src/go/go.mod

```diff
@@ -25,6 +25,7 @@ require (
 
 require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/getsentry/sentry-go v0.11.0 // indirect
```

Comment:
my `go mod tidy` run moves this into the direct block (we do reference it directly in `src/go/reporter/sentry/sentry.go`)

Reply:
ah that's on me - to implement sentry i go getted it will update